thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -140,21 +140,46 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
              */
             dedupList.sort(*iCompare, 1);
```

Comment:
I asked because I was wondering what happens to best conditions with duplicate values - but if not stable then they could be reordered anyway.